package.json

```diff
@@ -72,6 +72,7 @@
     "js-yaml": "3.12.0",
     "json-beautify": "1.0.1",
     "lodash": "4.17.11",
+    "logfmt": "^1.2.1",
```

Comment:
Can you provide some information on how and why this library is used? If I understand correctly it's also used in jaeger, but I'm not sure if we really need it, what I can see is that `searchOptions.tags` is expected to be a string in "logfmt" format and then is converted to plain json string to pass to jaeger, but what I fail to see is where these tags come from and if we can avoid having them in that format in the first place.

Also, I think you should remove the caret in lib version, as we don't use it in general.

Reply:
logfmt is used in src/actions/JaegerThunkActions.ts

Reply:
@aljesusg can you also fix that in your new PR? (the caret version)